library/bignum_mod_raw.c

```diff
@@ -119,7 +119,16 @@ int mbedtls_mpi_mod_raw_write( const mbedtls_mpi_uint *A,
 /* END MERGE SLOT 4 */
 
 /* BEGIN MERGE SLOT 5 */
-
+void mbedtls_mpi_mod_raw_add( mbedtls_mpi_uint *X,
+                              mbedtls_mpi_uint const *A,
+                              mbedtls_mpi_uint const *B,
+                              const mbedtls_mpi_mod_modulus *N )
+{
+    size_t carry, borrow = 0;
```

Comment:
these should be
```suggestion
    mbedtls_mpi_uint carry, borrow;
```

(and no point initialising just `borrow` to 0)